doc/modules/svm.rst

```diff
@@ -336,15 +336,8 @@ floating point values instead of integer values::
 Density estimation, novelty detection
 =======================================
 
-One-class SVM is used for novelty detection, that is, given a set of
-samples, it will detect the soft boundary of that set so as to
-classify new points as belonging to that set or not. The class that
-implements this is called :class:`OneClassSVM`.
-
-In this case, as it is a type of unsupervised learning, the fit method
-will only take as input an array X, as there are no class labels.
-
-See, section :ref:`outlier_detection` for more details on this usage.
+-See, section :ref:`outlier_detection` for more details on this usage.	
```

Comment:
You have a - at the start of the line.
I also think we should more specifically reference the documentation on svm there